rasa/utils/tensorflow/rasa_layers.py

```diff
@@ -128,7 +128,7 @@ def _replace_dense_for_sparse_layer(
         kernel = layer_to_replace.get_kernel().numpy()
         bias = layer_to_replace.get_bias()
         use_bias = False if bias is None else True
-        if use_bias:
+        if use_bias and bias is not None:
```

Comment:
given `use_bias` definition at the line above, this line is duplicating the checks. Removing the `use_bias` variable and converting it to `bias is not None` everywhere in this function seems like the way to go